KeenClient/KIODBStore.m

```diff
@@ -50,17 +54,27 @@ @implementation KIODBStore {
 - (instancetype)init {
     self = [super init];
 
-    if(self) {
+    if (nil != self) {
```

Comment:
Here and below for the `openLock` nil check, in general is the guidance that the explicit `nil` is necessary or is it OK to assume it's false-y? PR #211 seemed to have examples of both:

Favoring false-y `nil`:
```
 -    if ([HTTPCodes httpCodeType:(responseCode)] == HTTPCode4XXClientError && query != nil) {
 +    if (query && [HTTPCodes httpCodeType:(responseCode)] == HTTPCode4XXClientError) {
          // log what happened
```
...and favoring explicit `nil` check:
```
 -    if (![KIOQuery validateQueryType:queryType]) {
 +    if (queryType == nil || queryType.length <= 0) {
          return nil;
      }
```